flink-end-to-end-tests/test-scripts/common_s3_minio.sh

```diff
@@ -22,11 +22,11 @@ export AWS_REGION=us-east-1
 export AWS_ACCESS_KEY_ID=access_key
 export AWS_SECRET_ACCESS_KEY=secret_key
 
-export IT_CASE_S3_BUCKET=test-data
+IT_CASE_S3_BUCKET=test-data
 
-export S3_TEST_DATA_WORDS_URI="s3://$IT_CASE_S3_BUCKET/words"
-
-export S3_OUTPUT_LOCATION="s3://$(basename "$TEST_DATA_DIR")/out/wc_out"
+S3_TEST_DATA_WORDS_URI="s3://$IT_CASE_S3_BUCKET/words"
+OUTPUT_PATH="$TEST_INFRA_DIR/$IT_CASE_S3_BUCKET/$OUT"
```

Comment:
This script has no business deciding where data should be written to and should not write to TEST_INFRA_DIR in the first place but TEST_DATA_DIR as the latter is already scoped by test execution

Reply:
Moved it into the tests.